test/functional/api/v1/profiles_controller_test.rb

```diff
@@ -47,7 +47,7 @@ def response_body
       context "on GET to show when hide email" do
         setup do
           @user.update(hide_email: true)
-          get :show, id: @user.handle, format: format
+          get :show, params: { id: @user.handle }, format: format
```

Comment:
Changes like this should be done in a follow-up PR, in my opinion. This adds unnecessary review overhead to the Pull Request.